host/azure-pipelines.yml

```diff
@@ -15,31 +15,111 @@ pr:
     include:
       - host/src/
 
-pool:
-  vmImage: windows-latest
-
-steps:
-  # Remember to add this task to allow vcpkg to upload archives via NuGet
-  - task: NuGetAuthenticate@1
-
-  - task: DotNetCoreCLI@2
-    displayName: 'Publish win-x64 FunctionsNetHost'
-    inputs:
-      command: 'publish'
-      publishWebProjects: false
-      zipAfterPublish: false
-      arguments: '-c Release -r win-x64 -o $(Build.SourcesDirectory)/host/build/win-x64/FunctionsNetHost'
-      workingDirectory: $(Build.SourcesDirectory)/host/src/FunctionsNetHost
-
-  - task: NuGetCommand@2
-    displayName: "Nuget pack"
-    inputs:
-      command: "pack"
-      packagesToPack: "$(Build.SourcesDirectory)/host/tools/build/Microsoft.Azure.Functions.DotnetIsolatedNativeHost.nuspec"
-      versioningScheme: "off"
-      packDestination: "$(Build.ArtifactStagingDirectory)/host/build/nuget"
-      basePath: "$(Build.SourcesDirectory)/host/tools/build"
-
-  # Publish artifacts.
-  - publish: $(Build.ArtifactStagingDirectory)/host/build/nuget
-    artifact: drop
+stages:
```

Comment:
Example build using this pipeline: https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=133946&view=results